forem/forem
# added ability to assign moderator role via admin/super admin

## What type of PR is this? (check all applicable)


## Description

This PR allows a super admin to assign the moderator role to a user. NOTE: this does NOT enforce the moderator role, yet.

## Related Tickets & Documents

- Closes #17458 

## QA Instructions, Screenshots, Recordings

Enable the `moderator_role` feature flag and verify that the role is selectable in the roles drop down (special roles).
Assign a user the moderator role and ensure the moderator role is displayed when managing/editing that user.
_Note: When running Cypress tests, choose 'y' to rebuild the test database, so the test-moderator role is created._

### UI accessibility concerns?

no -- only a backend change

## Added/updated tests?


## [Forem core team only] How will this change be communicated?
